### ISSUE TITLE
Identifying aggregations of period concepts

### ISSUE THREAD

**Issue by rybesh** [periodo/periodo-data#40]
We currently group period concepts into the same set if they share a common published source. We identify the set as a "concept scheme," which means nothing more than "an aggregation of concepts." But there are reasons other than sharing a source that we might want to aggregate period concepts. People using PeriodO to provide a period authority file for their own system may want to aggregate a selection of other people's concepts. This requires being able to aggregate period concepts into a scheme and to assign the scheme a long-term identifier. Even in cases when the users of the authority file are also the authors, we need to distinguish the two aggregations: Pleiades' "currently preferred" period authority file differs from the "authored by Pleiades" aggregation, as the latter may include deprecated period concepts.

So, we need to allow period concepts to be in multiple schemes. This would not be a problem except that currently we connect the bibliographic description of the published source to the concept scheme (under the assumption that the periods in a scheme all share the same source). If we allow concepts to belong to multiple schemes, we need to allow a scheme to contain concepts from different sources. This means we ought to attach the bibliographic description of the published source to the concept as well. This further implies that we need to assign URIs to "our" (i.e. not from Crossref or Worldcat) bibliographic descriptions, since now they may be objects of multiple statements (`scheme -> source -> description` and `concept -> source -> description`). We could use fragment identifiers for these, e.g. `http://n2t.net/ark:/99152/p0fp7wv#source`.

Question: are aggregations other than "same source" aggregations part of the main dataset? Do curators need to accept patches to create new aggregations? Remember that we are giving these stable URIs, which means history tracking, etc. too. So we shouldn't enter that commitment lightly. On the other hand, it does seem that we need a way to create shared, persistently identified schemes, otherwise people will just update a local copy and will have no incentive to keep the canonical dataset up to date.

To recap:

* allow concepts to be in multiple schemes 
* attach source statements to individual period concepts 
* stop using blank nodes as objects of source statements 
* allow people to add and edit collections?

**Comment by rybesh** [comment]
Via Lex Berman:

> One followup on our conversation:  when Linda Hill first described to me her use of "broader" and "narrower" relations in Alexandria Feature Type Thesarus, it was pretty clearly neither spatial nor temporal, but purely semantic in the sense of 'superordinate' and 'subordinate.'
> 
> That's why I think in PeriodO case, if you have 12 items that each define the "Bronze Age," and you want to CONFLATE only three of them into your own concept of the Bronze Age, then you have:
> 
> Bronze Age (new conceptual container)
> conflates - Bronze Age (item1)
> conflates - Bronze Age (item2)
> conflates - Bronze Age (item3)
> 
> however it does not work in the thesaurus sense:
> 
> Bronze Age (new conceptual container)
> NT - Bronze Age (item1)
> NT - Bronze Age (item2)
> NT - Bronze Age (item3)
> 
> All of those items are peers.   We are conflating them into a conceptual container, but we are not able to use the temporal span of the subordinate items and say that the super-ordinate Bronze Age is a "broader term" because it's not semantically correct.   It IS a conflationOf them, though, so I think you need to mint your own relationship definition, unless you can find one for this.

**Comment by rybesh** [comment]
Summary of above: A narrower spatiotemporal footprint does not imply a subordinate concept. Whatever relation we end up using must not imply `skos:narrower`.

**Comment by rybesh** [comment]
With the merge of https://github.com/periodo/periodo-server/pull/65 we now have the ability to create and identify arbitrary sets of period definitions. We have decided **not** to implement any “conflated” concepts in the dataset or via the server. Instead, one will create a “bag” of definitions via the server and obtain a stable ID for that bag, which can be passed to local functions for conflating them in whatever way is desired.
